unlock-protocol.com/blog/nft-dev-meetup.md

```diff
@@ -0,0 +1,24 @@
+---
+title: NFT Dev Meetup
+subTitle: Livestream of the NFT Dev Meetup
+authorName: Sascha Mombartz
+publishDate: May 16, 2019
+description: More info and a livestream for registred participants.
+image: /static/images/blog/nftdevmeetup/hero.jpg
+---
+![Optimistic Unlocking in action.](/static/images/blog/nftdevmeetup/hero.jpg)
+
+We’re doing something different! By now you’ve probably been to a few too many after-parties. Let's not even talk about Consensys - did you see how packed the hallways were?
```

Comment:
```suggestion
We’re doing something different! By now you’ve probably been to a few too many after-parties. Let's not even talk about Consensus - did you see how packed the hallways were?
```

This should be Consensus not Consensys: a typo fix on this elsewhere was erroneous and will be changed back.